test/lib/code_corps/policy/task_skill_test.exs

```diff
@@ -13,49 +12,49 @@ defmodule CodeCorps.Policy.TaskSkillTest do
       user = insert(:user)
       task = insert(:task)
 
-      changeset = %TaskSkill{} |> create_changeset(%{task_id: task.id})
-      refute create?(user, changeset)
+      task_skill = %TaskSkill{task_id: task.id}
+      refute create?(user, task_skill)
```

Comment:
In "real-life", this function is being called from `CodeCorps.Policy` using a params map (with string keys) as the second argument.

However, the tests call it with a `%TaskSkill{}` with atom keys as the second argument.

The real life part is indirectly tested from the controller tests, and the tests you wrote here still pass, becase in `CodeCorps.Policy.Helpers`, we have

```Elixir
def get_task(%TaskSkill{task_id: task_id}), do: Repo.get(Task, task_id)
def get_task(%{"task_id" => task_id}), do: Repo.get(Task, task_id)
```

However, these tests leave room for a potential bug being introduced, so I would replace the second argument in `create?` tests with a params map. Note that the `delete?` tests do correctly use a `%TaskSkill{}` as the second argument, so there isn't anything to change there.